libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkHttpClient.cs

```diff
@@ -240,11 +227,9 @@ private async Task<InvokeResponse<T>> SecurePostActivityAsync<T>(Uri toUrl, Acti
                     using (var response = await HttpClient.SendAsync(httpRequestMessage, cancellationToken).ConfigureAwait(false))
                     {
                         var content = response.Content != null ? await response.Content.ReadAsStringAsync().ConfigureAwait(false) : null;
-                        return new InvokeResponse<T>
-                        {
-                            Status = (int)response.StatusCode,
-                            Body = content?.Length > 0 ? GetBodyContent<T>(content) : default
-                        };
+                        var invokeResponse = JsonConvert.DeserializeObject<InvokeResponse<T>>(content ?? "{}");
```

Comment:
Don't love the '{}' but like the one liner vs 5 liner.

Reply:
I suppose this captures the intent more obviously  
\`\`\`

var invokeResponse = (!String.IsNullOrEmpty(content)) ? JsonConvert.DeserializeObject&lt;InvokeResponse&lt;T&gt;&gt;(content) : new InvokeResponse&lt;T&gt;();  
\`\`\`